packages/framework/src/Component/Doctrine/Exception/DuplicatedAliasException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Doctrine\Exception;
+
+use LogicException;
+use Throwable;
+
+class DuplicatedAliasException extends LogicException
+{
+    /**
+     * @param string $alias
+     * @param int|null $code
+     * @param \Throwable|null $previous
+     */
+    public function __construct(string $alias, ?int $code = 0, ?Throwable $previous = null)
+    {
+        $message = sprintf(
+            'Alias "%s" is already assigned to different entity.',
+            $alias
+        );
+        parent::__construct(
+            $message,
```

Comment:
Personally, I would prefer to have the sprintf on one line here, but as it's only a matter of personal preference I leave it up to you if you change it or not.

Reply:
Usually I would do it the way you are describing. I did not because the sprintf method is kind a large (and I personally do not like multiline function as an argument) and maybe will be even larger as you are suggesting change the message.

Reply:
👌